java/HealthVault Mobile/HVSample/src/main/res/drawable-hdpi/circle_background.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="oval">
+    <solid android:color="#86bbbf"/>
+    <stroke android:width="2dp" android:color="#ffffff" />
+    <size android:width="180dp" android:height="180dp"/>
```

Comment:
We are using 180dp for a number of values in these resources which seem related. Lets be mindful and define a common resource in dimens.xml and reference it here. Same goes for the colors